static/app/components/dashboards/widgetQueriesForm.tsx

```diff
@@ -45,11 +45,21 @@ type Props = {
   fieldOptions: ReturnType<typeof generateFieldOptions>;
 };
 
+type State = {
+  blurTimeout?: ReturnType<typeof setTimeout>;
+};
+
 /**
  * Contain widget queries interactions and signal changes via the onChange
  * callback. This component's state should live in the parent.
  */
-class WidgetQueriesForm extends React.Component<Props> {
+class WidgetQueriesForm extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      blurTimeout: undefined,
+    };
+  }
```

Comment:
You don’t need to put this in a constructor. The state property can just be assigned in the class body

Reply:
👍 good call updated!